### PR TITLE
Changes related to external Population-Based Training integration

### PR DESCRIPTION
**Pull request by alex-petrenko**

### CODE REVIEW

**Comment by alex-petrenko** [comment]
@ViktorM FYI!

File: rl_games/common/a2c_common.py [hunk 1]
```diff
@@ -36,9 +38,90 @@ def rescale_actions(low, high, action):
     return scaled_action
 
 
+class IntervalSummaryWriter:
```
Comment (Denys88) [inline rl_games/common/a2c_common.py:41]: Could you move this class to the separate file?

File: rl_games/common/a2c_common.py [hunk 2]
```diff
@@ -36,9 +38,90 @@ def rescale_actions(low, high, action):
...
+            self.last_write_for_tag[tag] = time.time()
+
+    def __getattr__(self, attr):
+        return getattr(self.writer, attr)
+
+
 class A2CBase:
     def __init__(self, base_name, config):
+        pbt_str = ''
+        if config['pbt']:
```
Comment (Denys88) [inline rl_games/common/a2c_common.py:107]: maybe rename 'pbt' to the 'population_based_training' will easier to remember what is it in future

**Comment by alex-petrenko** [comment]
@Denys88 doen

**Comment by Denys88** [comment]
@alex-petrenko  LGTM

**Comment by alex-petrenko** [comment]
I guess the build problems are not related to the changes.
Let me know if any issues arise!
@ViktorM is in the loop on this too :)

**Comment by Denys88** [comment]
@alex-petrenko  thats true. For some reason it doesn't call pip isntall for dependencies. I am merging it to the master then.

**Pull request merged**